### PR TITLE
Development_Setup.rst: Fix link

### PR DESCRIPTION
**Pull request by yashtrivedi96**
Fix newcomers' guide link in
the Repositories section on line
23

Closes https://github.com/coala/coala/issues/3716

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by yash-nisar** [comment]
ack 9479b60c686c9ce9b90d800fb5253b4a2e7cbd7b

**Comment by adtac** [comment]
marking as wip till you rebase

**Comment by yashtrivedi96** [comment]
I have rebase please check

**Comment by adtac** [comment]
@yashtrivedi96 can you rebase once more? I just merged something else, so you're behind again.

**Comment by yashtrivedi96** [comment]
ok sure

**Comment by adtac** [comment]
ack https://github.com/coala/coala/pull/3735/commits/2cf78a504e70ceac34f7ddc9d6ec939ea8c7abc3

**Comment by supergr35** [comment]
In the future please develop on a different branch other than master, see our git documentation for more details :)

**Comment by yashtrivedi96** [comment]
Yes, to create a branch first :)

File: docs/Developers/Development_Setup.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Repositories
 ------------
 
 If you are interested in contributing to coala, we recommend that you read
-our :ref:`newcomers' guide <newcomer-guide>` to familiarize yourself with our
+our `newcomers' guide <http://api.coala.io/en/latest/Developers/Newcomers_Guide.html>`__ to familiarize yourself with our
```
Comment (Udayan12167) [inline docs/Developers/Development_Setup.rst]: Can you break the line after the link.

**Comment by Udayan12167** [comment]
unack 2cf78a5

**Comment by yashtrivedi96** [comment]
Ok

**Comment by yashtrivedi96** [comment]
@Udayan12167 is it proper?

**Comment by Udayan12167** [comment]
ack f9f22cf

**Comment by Udayan12167** [comment]
@rultor merge

**Pull request merged**